src/Field.jsx

```diff
@@ -327,9 +329,7 @@ class Field extends React.Component {
 
   _inject(child, isActive) {
     return {
-      className: cn(child.props.className, {
-        [this.props.errorClass]: isActive
-      })
+      className: cn(child.props.className, isActive && this.props.errorClass)
```

Comment:
This leads to no `errorClass` getting added if `errorClass` is `null` or something.